### DISCUSSION TITLE
Any plans to port SevenTV to Safari?

### DISCUSSION THREAD

**Discussion by mmattbtw** [SevenTV/Extension#57] [Q&A]
I personally use Safari on my Macbook since Chrome eats up battery :p 
Is there any plans to bring over 7tv to Safari?

From Apple on converting an existing Web Extension for Safari: https://developer.apple.com/documentation/safariservices/safari_web_extensions/converting_a_web_extension_for_safari

**Comment by miguelgargallo** [comment]
It should be, is a major browser on all macOS systems

**Comment by tranefrancks** [comment]
I'd like to add my vote here. 7TV extension on Safari would be very much welcome. I'd really like to not need to use a different browser just for Twitch.

**Reply by KZeni** [reply]
Not to forget that this would open up the door for using 7tv on iOS & iPadOS as well as macOS (ex. I use the same Safari ad-blocker on Mac & iOS [also then syncing preferences across devices, etc.])
